js/index.js

```diff
@@ -1354,8 +1357,10 @@ function DataManager() {
 			});
 		} else {
 			oTable.fnClearTable();
-			oTable.fnAddData(dataList);
-			oTable.fnAdjustColumnSizing();
+			if (dataList.length > 0){
+				oTable.fnAddData(dataList);
+				oTable.fnAdjustColumnSizing();
+			}
```

Comment:
At the beginning, the error happened [in this line](https://github.com/EyeSeeTea/datadictionary/blob/5dd62463752563bcef7225460eddf04498db8d91/js/utils/dhisDimensionUtils.js#L285). 'full' was an empty array. I fixed it there but then more errors showed up here and there. So instead of adding validations everywhere I decided to check it here. Maybe we can get adjustcolumnsizing out of the 'if'. Not sure if it would be useful.